src/main/java/org/sagebionetworks/web/client/widget/header/HeaderViewImpl.java

```diff
@@ -210,5 +227,6 @@ public void setPortalAlertVisible(boolean visible, JSONObjectAdapter json) {
   @Override
   public void setNIHAlertVisible(boolean visible) {
     nihNotificationAlert.setVisible(visible);
+    donationBannerContainer.setVisible(!visible);
```

Comment:
This approach does not scale easily to 3+ banners, but we recognize that the Home Page is about to have a redesign (so I don't want to invest in too much at this time)